docs/ref/contrib/gis/gdal.txt

```diff
@@ -1172,7 +1172,7 @@ blue.
             >>> rst = GDALRaster({'driver': 'GTiff', 'name': rstfile.name, 'srid': 4326,
             ...                   'width': 255, 'height': 255, 'nr_of_bands': 1})
             >>> rst.name
-            '/tmp/tmp7x9H4J.tif'           # The exact filename will be different on your computer
```

Comment:
I'm not so sure that `GDALRaster` doesn't store its files in `tmp` - do you want to check this?